src/ontology/mondo-edit.obo

```diff
@@ -123769,13 +123767,16 @@ xref: ICD10:A81.01 {source="EFO:1001233", source="MONDO:equivalentTo"}
 xref: ICD9:046.11 {source="EFO:1001233", source="MONDO:equivalentTo", source="i2s"}
 xref: MedDRA:10064199 {source="EFO:1001233"}
 xref: MESH:D016643 {source="EFO:1001233", source="DOID:5435", source="MONDO:directSiblingOf"}
+xref: MONDO:0005409 {source="MONDO:obsoleteEquivalent"}
```

Comment:
did the script introduce this? We should fix

Reply:
I'm not sure I understand what part you think the script introduced or what needs to be fixed?

Reply:
look on gh not in mail client. mondo shouldn't xref itself. Don't worry about it now but we should implement a check

Reply:
I thought we were meant to xref the term that was obsoleted, this is in the editors instructions (https://mondo.readthedocs.io/en/latest/editors-guide/merging-and-obsoleting/), see this example:

![image](https://user-images.githubusercontent.com/6722114/66089381-e3d52400-e533-11e9-8e50-63912f8991dc.png)

Is that what you meant? Or I am still missing something?